styleguide/docs/code_conventions.md

```diff
@@ -42,8 +42,105 @@ Class names and hierarchy follow the [BEM (Block Element Modifier)](http://getbe
 ### Responsive implementation using Breakpoint-Sass
 All patterns in the AMA Style Guide are fully responsive. We use [Breakpoint-Sass](http://breakpoint-sass.com/) to manage our media queries. Breakpoint is fairly simple to use and has a very thorough [wiki](https://github.com/at-import/breakpoint/wiki) explaining its useage and capabilities.
 
-### Sass-Grid
-Some of our layouts currently utilize the [Sass-grid](https://github.com/digitaledgeit/sass-grid) grid system. Sass-grid is a very lightweight, flexbox-based grid system.
+### Layouts
+
+The layouts molecules are now deprecated. To give structure to organisms and templates, please follow the grid and columns method outlined below.
+
+
+### Grids and columns
+
+#### Basics
+
+To use columns to control a template or organism's structure, start by adding an element with a `.grid` or `.container-with-grid` class (we'll get to the difference a little later). This grid instantiates a flexbox wrapper for descendant elements.
+
+Then add elements with `.col-width-x` classes, where 'x' should be replaced by the number of columns you want the element to span. We use a 12 column grid, so for any given `grid` container, make sure that the `col-width-x` numbers add up to a total of 12.
+
+**Example:**
+
+```
+<a href="#" class="grid">
+  <div class="col-width-8">
+    {% include '09-text.twig' %}
+  </div>
+  <div class="col-width-4">
+    {% include '09-text.twig' %}
+  </div>
+</a>
+```
+
+The `col-width-x` classes include default left and right padding. They default to `width:100%` at our mobile breakpoint (in other words, multiple columns collapse into a single-column layout). If you need to specify different gutters, or use a different mobile behavior for your pattern (e.g. in order to use different column widths or breakpoints), apply the `@include grid()` and `@include grid-_unit--cols(x)` mixins to your named classes* instead of using the `.grid` and `.col-width-x` classes:
+
+
+```
+// GOOD:
+<a href="#" class="news-section">
+  <div class="news-section_text">
+    {% include '09-text.twig' %}
+  </div>
+  <div class="news-section_text-other">
+    {% include '09-text.twig' %}
+  </div>
+</a>
+.news-section { 
+  @include grid(); 
+}
+.news-section_text { 
+  @include grid-_unit-cols(8); 
```

Comment:
@chelsiejohnston 
Typo in all the grid mixin examples: it should be two underscores after `grid`: `grid__unit--cols(4)`. I'm seeing a dash and then an underscore -- I wonder if something autoformatted when you didn't want it to

Reply:
@rpama nah, Megh and I just thought that's what the mixin was. Thought it was weird. :( Lol...